llvm/test/CodeGen/AMDGPU/no_straddle_sub.ll

```diff
@@ -0,0 +1,131 @@
+;RUN: llc --amdgpu-prevent-half-cache-line-straddling -amdgpu-scalarize-global-loads=false -mtriple=amdgcn -mcpu=gfx1200 -mattr=+real-true16,dumpcode -verify-machineinstrs --filetype=obj < %s | llvm-objdump --triple=amdgcn --mcpu=gfx1200 -d  - > %t.dis
+;RUN: %python %p/has_cache_straddle.py %t.dis
+
```

Comment:
There are too many new tests and I don't understand what the point of them all are; they all seem to duplicate existing tests just with disassembly